### DISCUSSION TITLE
Issues in flashing au280 with the provided scripts

### DISCUSSION THREAD

**Discussion by marcomole00** [Xilinx/open-nic-shell#63] [Q&A]
Hi,
i'm trying to flash the bitstream to a au280 board using the program_fpga.sh script provided in the repo. The scripts invokes vivado to execute the program_fpga.tcl script but then it fails at line 37 `puts "HW device: $hw_device"` , because the variable `$hw_device` is not declared. I invoke the bash script as described in the script itself : `EXTENDED_DEVICE_BDF1=<EXTENDED_DEVICE_BDF1> program_fpga.sh BITSTREAM_PATH BOARD`.

Any help would be useful, thanks.

**Comment by cneely-amd** [comment]
Hi @marcomole00 

Which version of Vivado are you using when encountering this issue?  

Thanks,
--Chris

**Comment by marcomole00** [comment]
Hi @cneely-amd, I discovered what my problem was: the FPGA i was using was connected only with PCIe and not JTAG/USB (It's connected to a remote server). Nevertheless I managed to flash the bitstream with xbflash2.
